debian/clickhouse-server.install

```diff
@@ -3,4 +3,5 @@ usr/bin/clickhouse-copier
 usr/bin/clickhouse-report
 etc/clickhouse-server/config.xml
 etc/clickhouse-server/users.xml
+etc/clickhouse-spec/
```

Comment:
`etc/clickhouse-rest-apis/`

Reply:
Here I think it would be more appropriate to use clickhouse-spec as the path name, because it means all documents under the spec and not just rest-api

Reply:
Seems making sense